openstacksdk_plugin/utils.py

```diff
@@ -605,3 +597,17 @@ def merge_resource_config(resource_config, config):
     if all(item and isinstance(item, dict)
            for item in [resource_config, config]):
         resource_config.update(**config)
+
+
+def generate_attachment_volume_key(prefix, volume_id, server_id):
+    """
+    This method helps to generate attachment volume key which can be used as
+    runtime property when running attaching/detaching volume from/to server
+    :param str prefix: Any prefix that could be added to the the key
+    :param str volume_id: Unique volume id
+    :param str server_id: Unique server id
+    :return str: attachment volume key
+    """
+    if all([prefix, volume_id, server_id]):
+        return '{0}-{1}-{2}'.format(prefix, volume_id, server_id)
+    return None
```

Comment:
What happens if `None` is received.

Reply:
I updated it so that we can fallback to default key